### PR TITLE
WIP Feature/enhanced framework

### PR DESCRIPTION
**Pull request by mrapitis**
Introduction of the Enhanced Framework into the SDK, currently work in progress.  More details in regards to a summary, completed tasks, and code examples to follow.

### CODE REVIEW

File: sdl_android_lib/src/com/smartdevicelink/abstraction/constants/SDLVersion.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.smartdevicelink.abstraction.constants;
+
+public enum SDLVersion {
```
Comment (justinjdickow) [inline sdl_android_lib/src/com/smartdevicelink/abstraction/constants/SDLVersion.java]: We're not accepting anything that differentiates between Ford specific head units

**Comment by joeygrover** [comment]
@mrapitis Is this pull request still valid? Or will the new separate pull requests be taking its place?

**Comment by mrapitis** [comment]
Not valid, we can close this one out... should have a new one pending from ty.

**Comment by joeygrover** [comment]
Closing out due to newer PRs

**Pull request closed without merging**